build/ci-build.yml

```diff
@@ -0,0 +1,122 @@
+name: Build Arcus.EventGrid CI
+
```

Comment:
All feedback from https://github.com/arcus-azure/arcus.webapi/pull/60#pullrequestreview-274904856 applies here as well.

Reply:
Only remaining aspect would be PRs for all branches